src/bindings/xfconf/scripts/replace-system-xfconf.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
```

Comment:
Please add very short description (meta-data) what this script does. Move it to the scripts folder and install it, so that it is also available via `kdb replace-system-xfconf`.

Reply:
Should be done by 6506d0e23efa25047381ef0cd2afa5aed13da3f7 or before